android/src/main/java/com/reactnativerate/RNRateModule.java

```diff
@@ -0,0 +1,61 @@
+package com.reactnativerate;
+
+import android.app.Activity;
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.Callback;
+import com.google.android.play.core.review.ReviewInfo;
+import com.google.android.play.core.review.ReviewManager;
+import com.google.android.play.core.review.ReviewManagerFactory;
+import com.google.android.play.core.tasks.OnCompleteListener;
+import com.google.android.play.core.tasks.Task;
+
+
+
+public class RNRateModule extends ReactContextBaseJavaModule {
+
+    private final ReactApplicationContext reactContext;
+
+    public RNRateModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+        this.reactContext = reactContext;
+    }
+
+    @Override
+    public String getName() {
+        return "RNRate";
+    }
+
+    @ReactMethod
+    public void rate(ReadableMap options, final Callback callback) {
```

Comment:
why passed argument `options` is not used inside the function ?

Reply:
![Screen Shot 2020-11-17 at 10 20 03 AM](https://user-images.githubusercontent.com/12766071/99370870-811d2400-28be-11eb-8cd8-a872de6776bb.png)